api/proto/metrics/metrics.proto

```diff
@@ -468,6 +468,41 @@ message ZMetricVolume {
   uint64 freeBytes = 9;
 }
 
+// Metrics for a snapshot
+// When a snapshot is created, its disk space is initially shared between
+// the snapshot and the file system, and possibly with previous snapshots.
+// As the file system changes, disk space that was previously shared becomes
+// unique to the snapshot, and thus is counted in the snapshot's used property.
+// Additionally, deleting snapshots can increase the amount of disk space
+// unique to (and thus used by) other snapshots.
```

Comment:
Does deleting snapshots ever increase the amount of disk space reported for the volume itself (prior to the delete it shares some space with its snapshots which is why I am asking).

Reply:
No, but deleting snapshots may increase the size of other snapshots

Reply:
Might make sense to add that info to the comment.

Reply:
I updated a comment